test/uk/gov/hmrc/gform/services/NumberValidationSpec.scala

```diff
@@ -73,6 +73,16 @@ class NumberValidationSpec extends Spec {
     validate(`fieldValue - number`, rawDataFromBrowser).futureValue shouldBe ().valid
   }
 
+  "Number format" should "not consider commas when computing the size of the whole number" in new Test {
```

Comment:
I already asked in previous PR, but why we have two test to validate
`"+789,343.34"` and two test to validate `"-789,343.34"`?

Reply:
I do not like this way of test. I think [Table-driven property checks](http://www.scalatest.org/user_guide/table_driven_property_checks) would be much better approach for this particular usecase.